### ISSUE TITLE
Doom-one theme contrast

### ISSUE THREAD

**Issue by gilbertw1** [hlissner/emacs-doom-themes#99]
Hey,

I just updated doom-themes and I noticed that with the most recent commit to the doom-one theme (b83b2b5e8a4da14c62aad4ecd5c187873a3def2d) that the theme contrast has changed drastically. Was this intentional? ... I much prefer the way it was previously:

Before: http://drop.bryan.sh/Ibtnju3Q6N.png
After: http://drop.bryan.sh/uKXaediy1r.png

Thanks!

**Comment by wpcarro** [comment]
@gilbertw1 I just `M-x package upgrade-all` and now I regret it... Same results as your screenshot

**Comment by jjwest** [comment]
If you're using solaire-mode you can add a call to `(solaire-mode-swap-bg)` to restore it.

**Comment by hlissner** [comment]
Hey folks, I apologize for the sudden change. I swapped bg and bg-alt in doom-one, which is a change I've meant to do since I split `solaire-mode` into its own plugin. The way it was, the screenshots were misleading to users who didn't use `solaire-mode`.

As @jjwest mentioned, I've added the command `(solaire-mode-swap-bg)` to `solaire-mode`. Calling it after the theme is loaded should fix the problem.
